src/WebDav.Client/WebDavClient.cs

```diff
@@ -732,6 +733,28 @@ public async Task<WebDavResponse> Unlock(Uri requestUri, UnlockParameters parame
             return new WebDavResponse((int)response.StatusCode, response.ReasonPhrase);
         }
 
+        /// <summary>
+        /// Use WebDav SEARCH command.
+        /// </summary>
+        /// <param name="requestUri">Base URI.</param>
+        /// <param name="parameters">Set additional search parameters.</param>
+        /// <returns></returns>
+        public async Task<WebDavResponse> Search(Uri requestUri, SearchParameters parameters)
```

Comment:
Wouldn't it be better to return `PropfindResponse` class from method?

Reply:
Changed it to PropFindResponse